test/videoroom_web/controllers/room_json_test.exs

```diff
@@ -135,6 +137,42 @@ defmodule VideoroomWeb.RoomJsonTest do
     end
   end
 
+  describe "Notifier down" do
+    test "Can join room after notifier exits with normal", %{
+      conn: conn,
+      client: client
+    } do
+      room_service_pid = Process.whereis(Videoroom.RoomService)
+
+      state = :sys.get_state(room_service_pid)
+
+      {_jellyfish_address, notifier} = state.notifier
+
+      _ws_caller = :sys.get_state(notifier)[:caller_pid]
+
+      Process.alive?(notifier)
```

Comment:
Shouldn't we
```suggestion
     assert Process.alive?(notifier)
```
here and below as well ?

Reply:
Sorry, that is some leftovers that were used for debugging.